specification/appplatform/resource-manager/Microsoft.AppPlatform/preview/2020-11-01-preview/appplatform.json

```diff
@@ -4472,7 +4472,7 @@
       "required": true,
       "type": "string",
       "enum": [
-        "2020-07-01"
+        "2020-11-01-preview"
```

Comment:
Why is the old version removed? Should new SDK also support the old version since this version just add two optional properties to the monitoring model.

Reply:
should auto generated from HUB, let me check and back to u later.

Reply:
@weidongxu-microsoft , will this enum value be used in JAVA SDK generation? Talked to @ArcturusZhang offline, this value is ignored in GO SDK generation. It uses the `version` value in `info` section.

Reply:
I don't think this enum will take effect in Java. The version in `info.version` will be used.

I would suggest to refer to the common type definition for api-version (https://github.com/Azure/azure-rest-api-specs/blob/master/specification/common-types/resource-management/v2/types.json#L543-L550), instead of creating a new one.

Reply:
Sorry for late response, take some other services as example, some like bc take similar way as this PR, while some just has no such enum here.

So here I am not sure which is the best practice and how sdk or other tools consume this enum. As I understanding, if this file totally for 2020-11-01-preview, add other version may confused ?

Reply:
as we discussed offline, updated.